### DISCUSSION TITLE
improve glance card  show_last_changed

### DISCUSSION THREAD

**Discussion by apaperclip** [home-assistant/frontend#10663] [Dashboard]
Unless I'm doing something wrong I would like to request an improvement to the show_last_changed glance card option.

I have a dashboard on my desk that shows the state of a few entities and I would like to see how long they've been in that state. I've noticed the glance card show_last_changed value will become stale and not update with the current relative-time. However, the logbook card will have an accurate relative time, even without refreshing.

Can the glance card be updated so the relative-time is accurate like the logook card?

Below is a quick example I had handy. If I let it sit longer the logbook relative time would continue to update whereas the glance time would not,.

![image](https://user-images.githubusercontent.com/67401560/142690701-63fd49f5-35c7-4034-9cc4-afb0ec571049.png)

This was on the current version of chrome and the user profile feature automatically close connection feature disabled.

Thanks

**Comment by spacegaier** [comment]
That is actually a bug. I will create an issue out of this discussion and push a fix (https://github.com/home-assistant/frontend/pull/10666).
